dotcom-rendering/src/lib/useAuthStatus.ts

```diff
@@ -6,6 +6,20 @@ import {
 	getSignedInStatusWithOkta,
 } from './identity';
 
+export const useIsSignedIn = (): boolean | 'Pending' => {
+	const authStatus = useAuthStatus();
+	switch (authStatus.kind) {
+		case 'Pending':
+			return 'Pending';
+		case 'SignedInWithCookies':
+		case 'SignedInWithOkta':
+			return true;
+		case 'SignedOutWithCookies':
+		case 'SignedOutWithOkta':
+			return false;
+	}
+};
```

Comment:
Love to see it!

Worth adding a bit of JSDoc to help consumers understand what 'Pending' means, and that it will be always returned for the server-side render, for example?

Reply:
That would be helpful - I was trying to word a question about it but it was proving to be very inelegant!

Reply:
👍 comment added